man/systemd.environment-generator.xml

```diff
@@ -121,6 +121,90 @@
     </example>
   </refsect1>
 
+  <refsect1>
+    <title>Environment Generator Path</title>
+
+    <para>Environment generators are executed from a set of paths determined
+    during compilation, described in the two tables below. Environment
+    generators found in directories listed earlier override ones with the
+    same name in directories lower in the list.</para>
+
+    <para>When the variable
+    <varname>$SYSTEMD_ENVIRONMENT_GENERATOR_PATH</varname> is set, the contents
+    of this variable overrides the generator path. If
+    <varname>$SYSTEMD_ENVIRONMENT_GENERATOR_PATH</varname> ends with an empty
+    component (<literal>:</literal>), the usual environment generator path will
+    be appended to the contents of the variable.</para>
```

Comment:
Hmm, maybe say
```
The variable <varname>$SYSTEMD_ENVIRONMENT_GENERATOR_PATH</varname> may 
contain a list of paths, separated by colons (<literal>:</literal>). When set, if
the list ends with an empty component (<literal>...:</literal>), this list is
prepended to the usual set of of paths.
Otherwise, the specified list replaces the usual set of paths.
```